{"uid":"3118fcf16649fa47","name":"test_update_avatar[api-11]","fullName":"tests.endpoints.test_avatar#test_update_avatar","historyId":"a7a17e0538edcb7a0d7186c317ff99c0","time":{"start":1733072502573,"stop":1733072503864,"duration":1291},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.ApiClient.ApiClient object at 0x7f29aec895e0>\nx_task_value = 'api-11'\n\n    @allure.feature(\"User API\")\n    @allure.story(\"Update user avatar\")\n    @pytest.mark.parametrize(\"x_task_value\", [\"api-11\"])\n    #@pytest.mark.skip(\"CI/CD issues\")\n    def test_update_avatar(api_client, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_avatar.py:18: ValidationError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1733072493396,"stop":1733072493397,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"api_client","time":{"start":1733072493398,"stop":1733072493398,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"setup_test","time":{"start":1733072493398,"stop":1733072495911,"duration":2513},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"base_url","time":{"start":1733072493398,"stop":1733072493398,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"worker_id","time":{"start":1733072493398,"stop":1733072493398,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.ApiClient.ApiClient object at 0x7f29aec895e0>\nx_task_value = 'api-11'\n\n    @allure.feature(\"User API\")\n    @allure.story(\"Update user avatar\")\n    @pytest.mark.parametrize(\"x_task_value\", [\"api-11\"])\n    #@pytest.mark.skip(\"CI/CD issues\")\n    def test_update_avatar(api_client, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_avatar.py:18: ValidationError","steps":[],"attachments":[{"uid":"595f11ea79e314df","name":"log","source":"595f11ea79e314df.txt","type":"text/plain","size":318},{"uid":"46b3dd834baa0fe4","name":"stderr","source":"46b3dd834baa0fe4.txt","type":"text/plain","size":314}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"setup_test::0","time":{"start":1733072505874,"stop":1733072505874,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"feature","value":"User API"},{"name":"story","value":"Update user avatar"},{"name":"parentSuite","value":"tests.endpoints"},{"name":"suite","value":"test_avatar"},{"name":"host","value":"fv-az849-971"},{"name":"thread","value":"2366-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.endpoints.test_avatar"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"x_task_value","value":"'api-11'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"3118fcf16649fa47.json","parameterValues":["'api-11'"]}